pkgs/development/web/spago/default.nix

```diff
@@ -0,0 +1,54 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
You should not import nixpkgs from within nixpkgs

Reply:
Yeah, I should've noticed earlier. It's fixed now, thanks!